content/reference/plugins/gitlab_ci/index.md

```diff
@@ -22,55 +32,195 @@ In the following examples, we assume the API Token is available in an environmen
 
 We also assume that you have already [configured a simulation]({{< ref "../../user/simulations" >}}) on Gatling Enterprise. You can copy the simulation ID from the simulations list view. In the following examples, we will show the simulation ID as `00000000-0000-0000-0000-000000000000`.
 
-## Using a build tool plugin
+## Quickstart (minimal pipeline, configuration)
 
-You can build you Simulation, and then run the updated Simulation on Gatling Enterprise, using the `enterpriseRun` command with any of our supported build tools.
+In this example, we configure a workflow which will only start a simulation as already configured and uploaded on Gatling Enterprise. We use the `workflow_dispatch` trigger event, so that we can [run it manually](https://docs.github.com/en/actions/managing-workflow-runs/manually-running-a-workflow), but feel free to use what works for your use case.
 
-With the `waitForRunEnd=true` option, it will display live metrics until the end of the run, and exit with an error code if the run fails on Gatling Enterprise (e.g. if the run crashes or if the assertions fail).
+Create a file named `.gitlab-ci.yml` in your repository:
 
-Configure your CI build to run the command corresponding to the build tool you use. Here are some examples:
+```yaml
+stages:
+  - load-test
 
-{{< include-file >}}
-Maven: includes/run-with-build-tool.maven.md
-Gradle: includes/run-with-build-tool.gradle.md
-Gradle Wrapper: includes/run-with-build-tool.gradlew.md
-Sbt: includes/run-with-build-tool.sbt.md
-{{< /include-file  >}}
+run-gatling-enterprise:
+  stage: load-test
+  image:
+    name: gatlingcorp/enterprise-runner:1
+    entrypoint: ['']
+  script:
+    - gatlingEnterpriseStart
+  variables:
+    # We assume GATLING_ENTERPRISE_API_TOKEN is available,
+    # e.g. configured on the GitLab project
+    SIMULATION_ID: '00000000-0000-0000-0000-000000000000'
+```
```

Comment:
why this one is not a full file?

Reply:
As far as I can tell, we didn't do it in all the other doc pages either. 🤷 (Except when there are tabs to chose between different formats, because that's how that component works.)